ParisBNF/abb/BNFabb106.xml

```diff
@@ -0,0 +1,226 @@
+                            <layoutDesc>
+                                <layout columns="3" writtenLines="24">   
+                                    <ab type="ruling" subtype="pattern" cert="medium">1A-1A-1A1A-1A1A/0-0/0-0/C</ab>
+                                    <ab type="ruling">Ruling is partly visible. </ab>
+                                    <ab type="pricking"> Pricking is partly visible.</ab>
+                                    <ab type="ruling" cert="unknown"> The upper line is written above the ruling. </ab>
+                                </layout>
+                            </layoutDesc>
+                        </objectDesc>
+                        <handDesc>
+                            <handNote xml:id="h1" script="Ethiopic">
+                                <seg type="script">19th-century script</seg> 
+                                <desc> <persName
+                                    ref="PRS1071dAbbadi">A. d'Abbadie</persName> describes the handwriting as ʻneatʼ<foreign xml:lang="fr">(ʻfineʼ)</foreign>.    
+                                </desc>
+                            </handNote>
+                        </handDesc>
+                            <additions>
+                            <list>
+                                <item xml:id="e1">
+                                    <desc type="StampExlibris">
+                                        Shelfmark of the <placeName ref="INS0303BNF">Library</placeName> glued onto the front board.
+                                </desc>
+                                </item>
+                                <item xml:id="e2">
+                                    <desc type="StampExlibris">
+                                        Shelfmark of the <placeName ref="INS0303BNF">Library</placeName> glued onto the inner side of the front board.
+                                    </desc>
+                                </item>
+                            </list>
+                            </additions>
+                        <bindingDesc>
+                            <binding  contemporary="true">
+                                <decoNote xml:id="b1" type="Boards">Two wooden boards.</decoNote>  
+                                <decoNote xml:id="b2" type="Cover">Leather quarter cover.</decoNote>  
+                                <decoNote xml:id="b3" type="SewingStations" cert="medium">4</decoNote>  
+                                <decoNote xml:id="b4" cert="medium">Two pairs of sewing stations.</decoNote>  
+                                <decoNote xml:id="b5" type="bindingMaterial">
+                                    <material key="wood"/>
+                                    <material key="leather"/>
+                                </decoNote>  
+                                <decoNote xml:id="b6" type="SlipCase">According to <persName
+                                    ref="PRS1071dAbbadi">A. d’Abbadie</persName>,
```

Comment:
also here "A." is not helpful, since there are several A. d'Abbadies around.
Either write out or do not fill the tag, `<persName ref="PRS1071dAbbadi"/>` is fully sufficient (and is recommended in most cases except when some particular wording is a necessary evidence; this is exactly why we have authority files, to avoid reproducing errors)

Reply:
Ok! Good to know! thank you!